spring-aop/src/main/java/org/springframework/aop/TargetClassAware.java

```diff
@@ -29,6 +29,7 @@
  * @since 2.0.3
  * @see org.springframework.aop.support.AopUtils#getTargetClass(Object)
  */
+@FunctionalInterface
```

Comment:
I am also thinking about this problem. It meets the requirements, but it will not be used with labmda. Should it be added the `@FunctionalInterface` annotation?